docs/server_guide.md

```diff
@@ -121,6 +121,7 @@ This document will introduce how to deploy Uniffle shuffle servers.
 | rss.storage.localFileWriterClass                         | org.apache.uniffle.storage.handler.impl.LocalFileWriter                | The writer class to write shuffle data for LOCALFILE.                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | rss.storage.hdfs.write.dataBufferSize                    | 8K                                                                     | The size of the buffer used to cache data written for HDFS.                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | rss.storage.hdfs.write.indexBufferSize                   | 8K                                                                     | The size of the buffer used to cache index written for HDFS.                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| rss.server.blockIdManagerClass                           | org.apache.uniffle.server.block.DefaultShuffleBlockIdManager           | The block id manager class. It is used to manage block id.                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
Could you add more description about defaultShuffleBlockIdManger?

Reply:
done